.bazelrc

```diff
@@ -3,6 +3,9 @@ build --action_env=BAZEL_CXXOPTS="-std=c++17:-fstrict-aliasing:-Wall:-Wsuggest-o
 build --copt=-fdiagnostics-color=always
 run --copt=-fdiagnostics-color=always
 test --copt=-fdiagnostics-color=always
+build --copt=-g
```

Comment:
Did you mean to leave this in?

Reply:
Yes, I did, it has low cost and I'm frequently annoyed by having to go turn it on to debug things